src/main/perl/Process.pm

```diff
@@ -723,8 +724,4 @@ it. But please, don't use it without a B<good> reason:
 
 It will only work with the C<execute> method.
 
-=head1 SEE ALSO
```

Comment:
Remark still valid if the code still wraps LC::Process?

Reply:
it is already mentioned (even in the description), and all LC::process inertaction is now part of a single method. i would also remove this section